test/foundry/scenarios/TokenSwaps.sol

```diff
@@ -98,82 +102,89 @@ contract TokenSwaps is ScenarioTestEnvironment {
 
-            _entryPoint.handleIntents(solution);
-        }
+        //execute
+        _entryPoint.handleIntents(solution);
 
         //verify end state
-        {
-            uint256 solverBalance = address(_publicAddressSolver).balance;
-            uint256 expectedSolverBalance = (erc20ReleaseAmount - ethRequireAmount) + slippage;
-            assertEq(solverBalance, expectedSolverBalance, "The solver ended up with incorrect balance");
-        }
-        {
-            uint256 userBalance = address(_account).balance;
-            uint256 expectedUserBalance = _accountInitialETHBalance + ethRequireAmount;
-            assertEq(userBalance, expectedUserBalance, "The user ended up with incorrect balance");
-        }
-        {
-            uint256 userERC20Tokens = _testERC20.balanceOf(address(_account));
-            uint256 expectedUserERC20Balance = _accountInitialERC20Balance - erc20ReleaseAmount;
-            assertEq(userERC20Tokens, expectedUserERC20Balance, "The user released more ERC20 tokens than expected");
-        }
+        uint256 solverBalance = address(_publicAddressSolver).balance;
+        uint256 expectedSolverBalance = (erc20ReleaseAmountEncoded - ethRequireAmountEncoded) + slippage;
+        assertEq(solverBalance, expectedSolverBalance, "The solver ended up with incorrect balance");
+
+        uint256 userBalance = address(_account).balance;
+        uint256 expectedUserBalance = _accountInitialETHBalance + ethRequireAmountEncoded;
+        assertEq(userBalance, expectedUserBalance, "The user ended up with incorrect balance");
+
+        uint256 userERC20Tokens = _testERC20.balanceOf(address(_account));
+        uint256 expectedUserERC20Balance = _accountInitialERC20Balance - erc20ReleaseAmountEncoded;
+        assertEq(userERC20Tokens, expectedUserERC20Balance, "The user released more ERC20 tokens than expected");
     }
 
-    function testFuzz_constantExpectation(
-        uint72 erc20ReleaseAmount,
-        uint72 ethRequireAmount,
-        uint16 releaseDuration,
-        uint16 releaseAt
-    ) public {
-        vm.assume(0 < erc20ReleaseAmount);
-        vm.assume(0 < releaseDuration);
-        vm.assume(0 < releaseAt);
-        vm.assume(releaseAt < releaseDuration);
-        vm.assume(ethRequireAmount < erc20ReleaseAmount);
-        vm.assume(erc20ReleaseAmount < _accountInitialERC20Balance);
+    function test_constantExpectation()
+        //uint72 erc20ReleaseAmount,
+        //uint72 ethRequireAmount,
+        //uint16 releaseDuration,
+        //uint16 releaseAt
+        public
+    {
+        //vm.assume(0 < erc20ReleaseAmount);
+        //vm.assume(0 < ethRequireAmount);
+        //vm.assume(0 < releaseDuration);
+        //vm.assume(0 < releaseAt);
+        //vm.assume(releaseAt < releaseDuration);
+        //vm.assume(ethRequireAmount < erc20ReleaseAmount);
+        //vm.assume(erc20ReleaseAmount < _accountInitialERC20Balance);
```

Comment:
```suggestion
```

Reply:
forgot to turn fuzzing back on when I was debugging